ReSharper.FSharp/src/FSharp.Psi.Daemon/src/QuickDoc/FSharpQuickDocProvider.fs

```diff
@@ -123,7 +117,12 @@ type FSharpQuickDocProvider(xmlDocService: FSharpXmlDocService) =
 
         context.GetData(PsiDataConstants.SELECTED_TREE_NODES)
         |> Seq.choose (function
-            | :? IFSharpIdentifier as node when node.GetSourceFile() = sourceFile -> Some node
+            | :? IFSharpIdentifier as node when node.GetSourceFile() = sourceFile ->
+                let activePatternCaseName = ActivePatternCaseNameNavigator.GetByIdentifier(node)
+                if isNotNull activePatternCaseName then
```

Comment:
If I remember correctly, the Navigators are allowed to take `null` as input.
So the `null` check should only happen for `activePatternId`.

Reply:
Right, thanks 👍